src/library_sdl.js

```diff
@@ -3007,8 +3012,11 @@ var LibrarySDL = {
     surfData.ctx.save();
     surfData.ctx.fillStyle = color;
     surfData.ctx.font = fontString;
-    surfData.ctx.textBaseline = 'top';
-    surfData.ctx.fillText(text, 0, 0);
+    // use bottom alligment, because it works 
+    // same in all browsers, more info here:
+    // https://bugzilla.mozilla.org/show_bug.cgi?id=737852
```

Comment:
Interesting. I know little about this too, but it does sound like your change is a good one.